src/components/DisplayFreeBond.tsx

```diff
@@ -0,0 +1,30 @@
+import { Box } from '@chakra-ui/react';
+import { useContractRead, useContractEvent, useAccount } from 'wagmi';
+import { ArchaeologistFacet__factory, ViewStateFacet__factory } from '../assets/typechain';
+
+function DisplayFreeBond() {
+  const { address } = useAccount();
+
+  const {
+    data: amount,
+    //    isError,
+    //    error,
+    refetch,
+  } = useContractRead({
+    addressOrName: process.env.REACT_APP_LOCAL_CONTRACT_ADDRESS || '',
```

Comment:
Is there a pattern we can use to handle testnet/mainnet addresses here as well as the local contract address?

Reply:
Yes. I can react to a network change and adjust the addresses accordingly. Can I add network switching as a task to the project board and leave this as is for now?

Reply:
@mobiusTripper-crypto -- yes, for sure! We will leave this as-is for this PR.